tests/scanner/scanner_test.py

```diff
@@ -32,6 +32,7 @@
 from google.cloud.security.scanner import scanner
 from google.cloud.security.scanner.audit import iam_rules_engine as ire
 from google.cloud.security.scanner.audit import rules as audit_rules
+from google.cloud.security.scanner.scanners import iam_rules_scanner as irs
```

Comment:
should we separate out the individual scanner tests into its own test suite and keep the scanner runner tests here?